### PR TITLE
Add some testunit

### PR DESCRIPTION
**Pull request by TTTaus**

### CODE REVIEW

File: src/test/java/systemtests/AddCommandSystemTest.java [hunk 1]
```diff
@@ -105,34 +105,7 @@ public void add() {
                 .withDepth(40)
                 .build();
         assertCommandFailure(DiveUtil.getAddCommand(diveSession), (Messages.MESSAGE_ERROR_LIMIT_EXCEED + " " + HARDCODE_MAX_TIME));
-
-        /*
-         * Case: Dive Depth is invalid - zero
-         */
-//        diveSession = new DiveSessionBuilder()
```
Comment (arjo129) [inline src/test/java/systemtests/AddCommandSystemTest.java]: Why is this removed!!!

**Comment by arjo129** [comment]
Why is it so hard for you to do things on time? Could this not have been done a week ago.

File: src/test/java/systemtests/AddCommandSystemTest.java (hunk 1 above)
Comment (TTTaus) [reply src/test/java/systemtests/AddCommandSystemTest.java]: Eh, I added that, but then there was other checks in place that would have made it harder thats why I inserted and then removed it
